src/webnn_native/NativeUtils.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WEBNN_NATIVE_NATIVEUTILS_H_
+#define WEBNN_NATIVE_NATIVEUTILS_H_
+
+#include <webnn/webnn_cpp.h>
+
+namespace webnn_native { namespace utils {
```

Comment:
To align with the namespace, could this file just be named as `Utils.h`?

Reply:
Thanks! I have renamed the file in this [commit](https://github.com/webmachinelearning/webnn-native/pull/165/commits/295c04658dacd36acbca90cb94d26fff6cac559e), PTAL. @huningxin